ksql-engine/src/test/java/io/confluent/ksql/topic/TopicCreateInjectorTest.java

```diff
@@ -425,6 +440,74 @@ public void shouldCreateMissingTopicWithDefaultCleanupPolicyForWindowedTables()
         ImmutableMap.of());
   }
 
+  @Test
+  public void shouldNotRegisterSchemaIfSchemaRegistryIsDisabled() {
+    // Given:
+    config = new KsqlConfig(ImmutableMap.of());
+    givenStatement("CREATE STREAM x (f1 VARCHAR) WITH(kafka_topic='expectedName', value_format='AVRO', partitions=1);");
+    when(builder.build()).thenReturn(new TopicProperties("expectedName", 10, (short) 10));
+
+    // When:
+    injector.inject(statement, builder);
+
+    // Then:
+    verifyNoMoreInteractions(schemaRegistryClient);
+  }
```

Comment:
I think this is missing some tests around what happens if schema registration fails...